build.gradle

```diff
@@ -354,21 +333,7 @@ def getGitVersion() {
     describeStdOut.toString().substring(1).trim()
 }
 
+apply from: 'gradle/javaToolchain.gradle'
 apply from: 'gradle/publish.gradle'
 apply from: 'gradle/deploy.gradle'
 apply from: 'gradle/javadoc.gradle'
-
-//////////////////////////////////////////
-// Root project configuration           //
-//////////////////////////////////////////
-wrapper {
-    gradleVersion = '6.0.1'
-}
-
-if (!JavaVersion.current().isJava9Compatible()) {
```

Comment:
We don't need this anymore?

Reply:
Because java toolchain functionality will error if the java version being used isn't compatible.